src/btree/bt_ret.c

```diff
@@ -135,7 +135,7 @@ __wt_read_cell_time_window(WT_CURSOR_BTREE *cbt, WT_TIME_WINDOW *tw, bool *tw_fo
         __read_col_time_window(session, page, WT_COL_PTR(page, &page->pg_var[cbt->slot]), tw);
         break;
     default: /* WT_PAGE_COL_FIX */
-        /* no time windows yet */
+        __wt_col_fix_fetch_time_window(session, cbt->ref, cbt->recno, tw, tw_foundp);
```

Comment:
Fixed. I'm going to leave off the default unless the compiler complains, because that makes this function need to be able to fail and we were talking below about changing it to return the found-flag.